index.html

```diff
@@ -46,146 +46,62 @@
 <body>
     <main>
         <section class="hero">
-            <h1><span class="hello1">"Hello! Sharing world"</span> <br>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
-            <p class="headline">A joyful celebration believing in the value of opennes and sharing,
-                creating a positive change with people from over 80 countries is taking place in
-                December, in México.
+            <h1><span class="hello1">"Hello! Real Estate world"</span> <br>REAL ESTATE SERVICES & PROFESSIONALS</h1>
+            <p class="headline">Pellentesque habitant morbi tristique senectus et netus et malesuada fames 
+                ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, 
+                ante. Donec eu libero sit amet quam egestas semper.
             </p>
             <h2 class="date">2022.12.15(THU) - 16(FRI)</h2>
             <p class="location">
-                @National park of México, México City
+                @Tlaxcala de Xicothencatl, Tlaxcala
             </p>
         </section>
         <section class="main-program" id="program">
-            <h3 class="sub-title">Main Program</h3>
+            <h3 class="sub-title">Main Services</h3>
             <hr class="line" width="50px">
             <div class="program-container">
                 <div class="program-div">
                     <img class="icon" src="./images/calendar_days_icon.svg" width="50px" alt="">
```

Comment:
- [ ] Kindly add alt to all the images so that other devices can access your site, devices like screen readers.